src/views/OpenOrders.vue

```diff
@@ -0,0 +1,220 @@
+              </ion-thumbnail>
+              <ion-label>
+                <p class="overline">WJ06-XL-PURPLE</p>
+                Juno Jacket
+                <p>Blue XL</p>
+              </ion-label>
+            </ion-item>
+          </div>
+          <div class="product-metadata">
+            <ion-note>49 pieces in stock</ion-note>
+          </div>
+        </div>
+
+        <div class="actions">  
+          <div class="positive-action"></div>
+          <div class="negative-action">
+            <ion-button class="desktop-only" fill="outline" color="danger">Recycle</ion-button>
+          </div>
+        </div>
+      </ion-card>
+
+      <ion-fab class="mobile-only" vertical="bottom" horizontal="end" slot="fixed">
+        <ion-fab-button @click="assignPickers">
+          <ion-icon :icon="print" />
+        </ion-fab-button>
+      </ion-fab> 
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import { IonButton, IonButtons, IonCard, IonChip, IonCheckbox, IonContent, IonFab, IonFabButton, IonHeader, IonLabel, IonIcon, IonItem, IonMenuButton, IonNote, IonPage, IonSearchbar, IonThumbnail, IonTitle, IonToolbar, modalController } from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { options, pricetag, print, refreshCircleOutline } from 'ionicons/icons';
+import AssignPickerModal from '@/views/AssignPickerModal.vue';
+
+export default defineComponent({
+  name: 'OpenOrders',
+  components: {
+    IonButton,
+    IonButtons,  
+    IonCard,
+    IonChip,
+    IonCheckbox,
+    IonContent,
+    IonFab,
+    IonFabButton,
+    IonHeader,
+    IonLabel,
+    IonIcon,
+    IonItem,
+    IonMenuButton,
+    IonNote,
+    IonPage,
+    IonSearchbar,
+    IonThumbnail,
+    IonTitle,
+    IonToolbar
+  },
+  methods: {
+    async assignPickers() {
+      const bgjobmodal = await modalController.create({
+        component: AssignPickerModal
+      });
+      return bgjobmodal.present();
+    },
+  },
+  setup() {
+    return{
+      options,
+      pricetag,
+      print,
+      refreshCircleOutline
+    }
+  }
+});
+</script>
+
+<style scoped>
+
+.filters > ion-item {
+ flex: 1 0 100%;
+ max-width: 200px;
+ border: 0.01px solid black;
+ border-radius: 10px;
+}
+
+.card-header {
+  display: grid;
+  grid: "tags"
+        "current"
+        "next"/ 1fr;
+  justify-content: start;
+  border-bottom: 1px solid black;        
+}
+
+.order-primary-info {
+  grid-area: current;
+}
+
+.order-tags {
+  grid-area: tags;
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid black;
+}
+
+.order-metadata {
+  grid-area: next;
+}  
+
+.order-item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}  
+
+ion-thumbnail> img {
+  object-fit: contain;
+}
+
+@media (min-width: 991px) {
+  .card-header {
+    grid: "current tags next" / max-content 1fr max-content;
+    justify-items: center;
```

Comment:
why are we justifying items to the center in desktop?

Reply:
Used justify self on child level.